src/notebook/rock-paper-scissors.ipynb

```diff
@@ -63,9 +63,9 @@
    "outputs": [],
    "source": [
     "# go to the folder corresponding to your object container\n",
-    "%cd /workspace/attendee\n",
+    "%cd /workspace/data\n",
     "\n",
-    "!curl -L \"https://storage.gra.cloud.ovh.net/v1/AUTH_2b59f4f857964cc28bb66c7155b6c8d9/workshop-resources/Rock-Paper-Scissors-SXSW.v12i.yolov8.zip\" > roboflow.zip; unzip roboflow.zip; rm roboflow.zip"
+    "!curl -L \"<dataset_url>\" > roboflow.zip; unzip roboflow.zip; rm roboflow.zip"
```

Comment:
same as for solution/ai/notebooks/YOLOV8/rock-paper-scissors.ipynb
Is it possible to use a variable for this <dataset_url>?

Reply:
yes I try to do the same thing that https://github.com/devrel-workshop/101-AI-and-py/pull/85#discussion_r1516174129